node/env0-deploy-utils.js

```diff
@@ -50,7 +50,7 @@ class DeployUtils {
     }
 
     console.log(`setting the following configuration: ${JSON.stringify(configuration)}`);
-    await apiClient.callApi('post', 'configuration', {data: configuration});
+    await apiClient.callApi('post', 'configuration', {data: {...configuration, projectId: undefined}});
```

Comment:
![bitmoji](https://render.bitstrips.com/v2/cpanel/dcbaa397-a792-45fa-8af9-8c55a26b5724-f93a000d-009c-474a-9778-5f01c11b88f9-v1.png?transparent=1&palette=1&width=246)

@onhate - I know that I wrote the 99% of the code here, and it's my fault, but @avnerenv0 started writing unit tests for this project, so let's try to add a specific unit test for each change as we go along with this project.

Reply:
done :)

Reply:
we probably need to configure github actions to run the tests now

Reply:
https://github.com/env0/env0-client-integrations/pull/9/checks?check_run_id=387570780